src/libexec/zmdbintegrityreport

```diff
@@ -132,15 +132,31 @@ sub checkDbs() {
       }
     }
   }
-  if (scalar @output != 0) {
-    addToReport("Database errors found.\n");
-    addToReport("$cmd --password=XXXXXXXX\n");
-    addToReport("@output");
-    if(!$options{m}) {
-      return 1;
-    }
+  my $mysql_cmd = "/opt/zimbra/common/bin/mysql -S ${mysql_socket} -u root --password=${mysql_root_passwd}";
+  my @mistmatch_records;
+  for my $num (1..100) {
+	  if (system("$mysql_cmd -e 'use mboxgroup$num' 2>/dev/null") == 0) {
+		  my $query = "select id from zimbra.mailbox where group_id=$num and id not in (select distinct(mailbox_id) from mboxgroup$num.mail_item)";
```

Comment:
can we add some handling here if the query fails we can abort? 
for e.g. like if there are only 70 mboxgroups and query is trying above that mboxgroups for 71

Reply:
It will continue if the mboxgroup database exists.